src/appleseed.cli/progresstilecallback.cpp

```diff
@@ -90,12 +102,28 @@ namespace
 
             // Print a progress message.
             LOG_INFO(m_logger, "rendering, %s done", pretty_percent(m_rendered_pixels, total_pixels).c_str());
+
+            // Keep track of the total number of rendered tiles.
+            m_rendered_tiles++;
+
+            // Retrieve the total number of tiles in the frame.
+            const size_t total_tiles = frame->image().properties().m_tile_count;
+
+            // Estimate remaining render time.
+            m_stopwatch.measure();
+            const double elapsed_time = m_stopwatch.get_seconds();
+            const double estimated_time = (elapsed_time / m_rendered_tiles) * (total_tiles - m_rendered_tiles);
+
+            // Print estimated time.
+            LOG_INFO(m_logger, "remaining time %.3lf seconds", estimated_time);
         }
 
       private:
         Logger&             m_logger;
         boost::mutex        m_mutex;
         size_t              m_rendered_pixels;
+        size_t              m_rendered_tiles;
+        Stopwatch<DefaultWallclockTimer>    m_stopwatch;
```

Comment:
Align members vertically.

You need to include `foundation/platform/defaulttimers.h` for `DefaultWallclockTimer`.

Reply:
sorry didn't see it! gonna include it in next commit

Reply:
done